documentation/tools/oracle-factory/quick-start.md

```diff
@@ -1,174 +1,88 @@
 # Quick Start
 
-## 1. Setup Environment
+## 1.  Getting started
 
-Ensure you have the necessary environment for running the Oracle Factory SDK.
+### **1.1.  Prerequisites**
 
-### **1.1. Prerequisites**
+Before getting started, ensure that you have the following installed on your system:
 
-Before starting, make sure you have the following installed:
+\- [**Node.js**](https://nodejs.org/en/) version 18 or higher
 
-- [**Node.js**](https://nodejs.org/en/) version 18 or higher
-- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)
+\- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)\
 
-## 2. Initialize Your Project
+### **1.2. Start a new project**
 
-### **2.1. Clone the Project**
+In this section, we will show you how to set up and call the different methods of the SDK in a React.js app.
 
-Start by cloning the Oracle Factory project repository. You can find the repository [here](https://github.com/iExecBlockchainComputing/iexec-oracle-factory-wrapper).
+You can reach the following open-source GitHub project, clone it and start from there.
 
-```bash
-git clone https://github.com/iExecBlockchainComputing/iexec-oracle-factory-wrapper
-cd iexec-oracle-factory-wrapper
-```
+You can find the project [here](https://github.com/iExecBlockchainComputing/iexec-oracle-factory-wrapper).
```

Comment:
il faut remplacer ce lien par un lien qui initialize un projet react avec oracel-factory-wrapper installé (projet sandbox)

=> créer un ticket sur Jira pour ajouter un exemple de of-wrapper sur sandbox (voir dataprotector et web3mail)